raiden/smart_contracts/ChannelManagerLibrary.sol

```diff
@@ -10,7 +10,7 @@ library ChannelManagerLibrary {
         address[] all_channels;
         Token token;
     }
-
+    event ChannelDelete(address caller_address, address partner, address channel_address);
```

Comment:
Please add new line before and after this line. It does not look good this way. And also please make each argument on its own line, like you did in ChannelManagerContract